crates/ruff_python_ast/tests/parenthesize.rs

```diff
@@ -10,7 +12,7 @@ fn test_parenthesized_name() {
     let name = bin_op.left.as_ref();
 
     let parenthesized = parenthesized_range(name.into(), bin_op.into(), source_code);
-    assert!(parenthesized.is_some());
+    assert_debug_snapshot!(parenthesized);
```

Comment:
Snapshot tests are neat for asserting against large and complicated data structure but have the downside that the expected value isn't visible inside the test itself. It requires future us to read the test, find the right snapshot file (snapshot file names are very long), and then compare the values. This is significantly more work compared to having the expected value inline and using `assert_eq!(parenthesized, Some(TextRange::new(x, y)))`

Reply:
Makes sense.